src/main/java/at/hannibal2/skyhanni/config/features/garden/cropmilestones/CropMilestonesConfig.java

```diff
@@ -89,6 +89,13 @@ public String toString() {
     @ConfigEditorBoolean
     public Property<Boolean> bestShowMaxedNeeded = Property.of(false);
 
+    @Expose
+    @ConfigOption(
+        name = "Overflow Milestones",
+        desc = "Show overflow crop milestones beyond 46.")
+    @ConfigEditorBoolean
+    public Property<Boolean> overflowMilestones = Property.of(false);
```

Comment:
```suggestion
    public boolean overflowMilestones = false;
```
Does not need to be a property